### DISCUSSION TITLE
HL1 Mods with Ray Tracing?

### DISCUSSION THREAD

**Discussion by nemix** [sultim-t/xash-rt#68] [General]
Can you play other HL1 Mods with this to get ray tracing for them?

**Comment by sultim-t** [comment]
If a mod requires a special logic to run (custom .dll files) then it needs to be recompiled for x64 and xash-compatible SDK.
If a mod adds custom maps, then it's also a problem, as tweaking the lights need to be done on each level, as default lights probably won't look fine.

**Reply by VanderCat** [reply]
i got a decent results by tweaking rt_emis_mapboost and rt_light_s

**Reply by Implicit-D** [reply]
> If a mod requires a special logic to run (custom .dll files) then it needs to be recompiled for x64 and xash-compatible SDK. If a mod adds custom maps, then it's also a problem, as tweaking the lights need to be done on each level, as default lights probably won't look fine.

I'm curious about support for the HD model pack that comes with the Steam release. I'm one of those heretics that actually kinda likes them better. It's a nostalgia thing. I grew up reading Gamepro and seeing the Dreamcast and PS2 port previews back in the day before I had a PC that could run Half-Life worth a damn. So of course I got the leaked DC HL1 as soon as I could, and I've always had a soft spot for the HD model pack.

**Reply by Implicit-D** [reply]
> If a mod requires a special logic to run (custom .dll files) then it needs to be recompiled for x64 and xash-compatible SDK. If a mod adds custom maps, then it's also a problem, as tweaking the lights need to be done on each level, as default lights probably won't look fine.

Not related, but Sultim... putting your signature on the wall in everyone's least favorite part of the game? Bold move lol.

**Comment by VanderCat** [comment]
to play simple hl1 mods you will need to copy `valve` folder in `Half-Life/rt` directory and rename it to the mod you want to play. You will also need to tweak light console commands and run with `-game MOD_NAME` cli argument

for example Hazardous Course 2 folder is `hc` so copy `valve` and rename it to `hc` then run xash with `-game hc`

My result:
https://imgur.com/a/bW3wIuj

**Reply by cambramCambram** [reply]
can you explain to me how to "run xash with -game hc"

**Reply by cambramCambram** [reply]
nevermind i figured it out you confused me with the - but i typed "game (mod)" in the console thank you for helping me put mods in rtx
